bigquery/samples/tests/test_dataset_samples.py

```diff
@@ -13,10 +13,52 @@
 # limitations under the License.
 
 from .. import create_dataset
+from .. import delete_dataset
+from .. import get_dataset
+from .. import list_datasets
+from .. import update_dataset_description
+from .. import update_dataset_default_table_expiration
+from .. import update_dataset_access
 
 
-def test_create_dataset(capsys, client, random_dataset_id):
+def test_dataset_samples(capsys, client, random_dataset_id):
 
+    # create dataset
     create_dataset.create_dataset(client, random_dataset_id)
     out, err = capsys.readouterr()
     assert "Created dataset {}".format(random_dataset_id) in out
+
+    # get dataset
+    get_dataset.get_dataset(client, random_dataset_id)
+    out, err = capsys.readouterr()
+    assert random_dataset_id in out
+
+    # list dataset
+    list_datasets.list_datasets(client)
+    out, err = capsys.readouterr()
+    assert "Datasets in project {}:".format(client.project) in out
+
+    # update dataset description
+    update_dataset_description.update_dataset_description(client, random_dataset_id)
+    out, err = capsys.readouterr()
+    assert "Updated description." in out
+
+    # update dataset table expiration
+    update_dataset_default_table_expiration.update_dataset_default_table_expiration(
+        client, random_dataset_id
+    )
+    out, err = capsys.readouterr()
+    assert "Updated dataset {}".format(random_dataset_id) in out
```

Comment:
Please look for the expected expiration `str(24 * 60 * 60 * 1000)` in the output too. That way we can be more certain that the expiration was actually updated as expected.

Reply:
Moved the time into it's own fixture so it's easier to be used in the test.